### PR TITLE
Standalone action tracker responses

### PR DESCRIPTION
**Pull request by Jhneric**
Standalone action tracker responses

### CODE REVIEW

**Comment by Jhneric** [comment]
Standalone action tracker Links

File: mkdocs.yml [hunk 1]
```diff
@@ -309,6 +309,16 @@ nav:
               - DHIS2 Frequently Asked Questions: '@github(dhis2/dhis2-docs, src/user/dhis2-frequently-asked-questions.md,master)'
               - Release and upgrade notes: '@github(dhis2/dhis2-docs, src/user/release-and-upgrade-notes.md,master)'
         Optional Apps:
+          Standalone Action Tracker App:
+            alt__App version 1.0:
+              - Standalone Action Tracker App Demo:
+                - Introduction: '@github(hisptz/unicef-apps-docs,src/commonmark/en/content/standalone_action_tracker/sat-app-introduction.md, master)'
+                - Introduction: '@github(hisptz/unicef-apps-docs,src/commonmark/en/content/standalone_action_tracker/sat-app-architecture.md, master)'
```
Comment (Philip-Larsen-Donnelly) [inline mkdocs.yml:316]: Seems to be a repetition of the above line

**Review by Philip-Larsen-Donnelly — CHANGES_REQUESTED** [review]
Build fails. If seems to be because the paths do not exist in the unicef-apps-docs repository

File: mkdocs.yml (hunk 1 above)
Comment (Jhneric) [reply mkdocs.yml]: Sent another PR. Please review

**Review by Philip-Larsen-Donnelly — APPROVED** [review]

**Pull request merged**